<launch>
  <arg name="smb_number" default="264"/>
  <node name="artefact_mapping" pkg="artefact_mapping" type="artefact_mapping" >
    <param name="object_tracker_image_topic" value="/versavis/cam0/image_raw"/>
    <param name="sensor_calibration_file" value="$(find artefact_mapping)/share/camchain-smb$(arg smb_number).yaml"/>
    <param name="darknet_cfg_path" value="$(find artefact_mapping)/share/yolov3.cfg"/>
    <param name="darknet_weights_path" value="$(find artefact_mapping)/share/yolov3.weights"/>
    <param name= "darknet_classes" value="46,25,24,62,47,0"/>
    <!-- human= 0; stop sign=11; backpack=24; umbrella=25; bottle=39; clock=74;-->
    <param name= "object_tracker_image_topic" value="/versavis/cam0/image_raw"/>
    <param name= "object_tracker_detection_period" value="10"/>
    <param name= "publish_debug_images" value="true"/>
    <param name= "v" value="1"/>
    <param name= "odom_tf_frame" value="/tracking_camera_odom"/>
    <param name= "map_tf_frame" value="/map"/> <!--for competition change this one to map instead of LH_WHEEL-->
    <param name= "darknet_detection_threshold" value="0.2"/>
    <param name="image_topic_buffer_size" value="1000"/>
    
  </node>  
</launch>
